prior_localization/prepare_data.py

```diff
@@ -206,7 +208,7 @@ def prepare_behavior(
     # load imposter trials dataframe if required
     if target in ['wheel-speed', 'wheel-velocity'] and np.any(np.array(pseudo_ids) > 0):
         assert config['imposter_df_path'] is not None, 'Must specify imposter_df_path in config.yaml file'
-        imposter_df = pd.read_parquet(config['imposter_df_path'])
+        imposter_df = pd.read_parquet(os.path.join(config['imposter_df_path'], f'imposterSessions_{target}.pqt'))
```

Comment:
Preferably use pathlib instead of os.path
```
from pathlib import Path
imposter_df = pd.read_parquet(Path(config['imposter_df_path']).joinpath(f'imposterSessions_{target}.pqt'))
```